modules/acquia_cms_common/composer.json

```diff
@@ -76,11 +76,7 @@
         "enable-patching": true,
         "patches": {
             "drupal/core": {
-                "3301692 - Passing null to parameter to mb_strtolower() is deprecated with PHP 8.1": "https://git.drupalcode.org/project/drupal/-/merge_requests/2598.patch",
-                "296693 - Restrict access to empty top level administration pages": "https://www.drupal.org/files/issues/2023-08-31/296693-10.1.x_0.patch",
-                "3371005 - Toolbar does not indicate active menu trail for pages not included in Toolbar": "https://git.drupalcode.org/project/drupal/-/commit/3165269bb01a5a8e5f53c1f369135b967c9d5924.patch",
-                "3370946 - Page title should contextualize the local navigation": "https://www.drupal.org/files/issues/2023-09-11/3370946-page-title-backport-10-1-x.patch",
```

Comment:
Unlike the other patches, this was not committed to Core yet.

Reply:
This is scheduled for release as part of ACMC Common 3.4.0, aligning with Drupal core version 10.2.0 stable release. We anticipate its inclusion in the core by that time."

Reply:
It's okay @timplunkett , we are removing these four patches and release the next module version once Drupal 10.2 is released. We have confirmation from @mikemadison13 that we have to remove all patches and expect the patch features to be part of Drupal 10.2 stable version.

Reply:
Added updated patch for Drupal 10.2.x version